--- a/lua/lv-utils/init.lua
+++ b/lua/lv-utils/init.lua
@@ -57,9 +57,8 @@ function lv_utils.generate_settings()
 end
 
 function lv_utils.reload_lv_config()
   vim.cmd "source ~/.config/nvim/lua/core/formatter.lua"
-  vim.cmd "source ~/.local/share/lunarvim/lvim/lua/settings.lua"
   vim.cmd "source ~/.local/share/lunarvim/lvim/lua/core/formatter.lua"
   vim.cmd ":PackerCompile"
   vim.cmd ":PackerInstall"
   -- vim.cmd ":PackerClean"
